addons/oceanbase-ce/templates/componentdefinition.yaml

```diff
@@ -1,215 +1,177 @@
-            - bash
-            - -c
-            - |
-              /scripts/setupmetrics.sh
-          env:
-            - name: OB_MONITOR_STATUS
-              value: active
-            - name: CLUSTER_NAME
-              value: "$(KB_CLUSTER_COMP_NAME)"
-            - name: CLUSTER_ID
-              value: "1"
-            - name: MONITOR_USER
-              value: "root"
-            - name: MONITOR_PASSWORD
-              value: ""
-            - name: SERVICE_PORT
-              value: "8088"
-            - name: OB_SERVICE_PORT
-              value: "2881"
-          ports:
-            - containerPort: 8088
-              name: http
-              protocol: TCP
-            - containerPort: 8089
-              name: pprof
-              protocol: TCP
-  #        readinessProbe:
-  #          failureThreshold: 3
-  #          httpGet:
-  #            path: /metrics/stat
-  #            port: 8088
-  #            scheme: HTTP
-  #          initialDelaySeconds: 5
-  #          periodSeconds: 2
-  #          successThreshold: 1
-  #          timeoutSeconds: 1
-          resources:
-            limits:
-              cpu: "1"
-              memory: 1Gi
-            requests:
-              cpu: "200m"
-              memory: 256Mi
-          terminationMessagePath: /dev/termination-log
-          terminationMessagePolicy: File
-          volumeMounts:
-            - name: scripts
-              mountPath: /scripts
-            - name: kb-tools
-              mountPath: /kb_tools
-            - name: metricslog
-              mountPath: /home/admin/obagent/log
-          workingDir: /home/admin/obagento
-      volumes:
-        - name: metricslog
-          emptyDir: {}
-- apiVersion: apps.kubeblocks.io/v1
-  kind: ComponentDefinition
-  metadata:
-    name: {{ printf "%s-repl" (include "oceanbase-release.name" .) }}
-    labels:
-      {{- include "oceanbase.labels" . | nindent 6 }}
-  spec:
-    provider: *obprovider
-    serviceKind: *obsvckind
-    serviceVersion: *obver
-    description: *obdesc
-    services: *observices
-    updateStrategy: Serial
-    configs: *obconfigs
-    scripts: *obscripts
-    roles: &obroles
-      - name: primary
-        serviceable: true
-        writable: true
-        votable: true
-      - name: standby
-        serviceable: true
-        writable: false
-        votable: true
-    lifecycleActions: &oblifecycle
-      roleProbe:
-        periodSeconds: {{ .Values.roleProbe.periodSeconds }}
-        timeoutSeconds: {{ .Values.roleProbe.timeoutSeconds }}
-        exec:
-          container: observer-container
-          command:
-            - /tools/dbctl
-            - --config-path
-            - /tools/config/dbctl/components
-            - oceanbase
-            - getrole
-    labels: &oblabels
-      kubeblocks.io/ready-without-primary: "true"
-    runtime: *obruntime
+  name: {{ include "oceanbase-ce.compDefName" . }}
+  labels:
+    {{- include "oceanbase-ce.labels" . | nindent 4 }}
```

Comment:
add the annotations for helm resource policy keep

Reply:
Fixed.